test/test_python_ex_standalone.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+import pytest
+
+from container_ci_suite.openshift import OpenShiftAPI
+from container_ci_suite.utils import get_service_image, check_variables
+
+if not check_variables():
+    print("At least one variable from IMAGE_NAME, OS, SINGLE_VERSION is missing.")
+    sys.exit(1)
+
+VERSION = os.getenv("SINGLE_VERSION")
+IMAGE_NAME = os.getenv("IMAGE_NAME")
+OS = os.getenv("TARGET")
+
+
+# Replacement with 'test_python_s2i_app_ex_standalone'
+class TestPythonExTemplate:
+
+    def setup_method(self):
+        self.oc_api = OpenShiftAPI(pod_name_prefix="python-testing", version=VERSION)
+
+    def teardown_method(self):
+        self.oc_api.delete_project()
+
+    def test_python_ex_template_inside_cluster(self):
+        service_name = "python-testing"
+        assert self.oc_api.deploy_s2i_app(
+            image_name=IMAGE_NAME, app=f"https://github.com/sclorg/s2i-python-container.git",
```

Comment:
F-string without any variable inside.

```suggestion
            image_name=IMAGE_NAME, app="https://github.com/sclorg/s2i-python-container.git",
```